hack/generate-swagger-models.sh

```diff
@@ -4,11 +4,59 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
+# Get the absolute path of this file
+DIR="$( cd "$( dirname "$0"  )" && pwd  )"
+cd "${DIR}" || return
+
 # Here is details of the swagger binary we use.
 # $ swagger version
 # version: 0.19.0
+SWAGGER_VERSION=v0.19.0
 
-# Get the absolute path of this file
-DIR="$( cd "$( dirname "$0"  )" && pwd  )"
+. ./env.sh
+
+# swagger::check_version checks the command and the version.
+swagger::check_version() {
+  local has_installed version
+
+  has_installed="$(command -v swagger || echo false)"
+  if [[ "${has_installed}" = "false" ]]; then
+    echo false
+    return
+  fi
+
+  version="$(swagger version | head -n 1 | cut -d " " -f 2)"
+  if [[ "${SWAGGER_VERSION}" != "${version}" ]]; then
+    echo false
+    return
+  fi
+  echo true
+}
+
+# swagger::install installs the swagger binary.
+swagger::install() {
+    has_installed="$(swagger::check_version)"
+    if [[ "${has_installed}" = "true" ]]; then
+        echo "swagger-${SWAGGER_VERSION} has been installed."
```

Comment:
LGTM, just some nits.
If I have installed swagger, it will echo
``` bash
swagger-v0.19.0 has been installed.
```
Looks like a little confused that if I have installed swagger before or just installed through this script.
Is it better to change it to `swagger-${SWAGGER_VERSION} has already been installed. Skip installation.`

Reply:
Updated.